 
 -------------- celery@jannone5 v5.3.6 (emerald-rush)
--- ***** ----- 
-- ******* ---- Linux-6.5.0-26-generic-x86_64-with-glibc2.38 2024-03-28 09:40:44
- *** --- * --- 
- ** ---------- [config]
- ** ---------- .> app:         truth_will_set_you_free:0x7b5e58782030
- ** ---------- .> transport:   redis://localhost:6379/0
- ** ---------- .> results:     redis://localhost:6379/0
- *** --- * --- .> concurrency: 20 (prefork)
-- ******* ---- .> task events: OFF (enable -E to monitor tasks in this worker)
--- ***** ----- 
 -------------- [queues]
                .> celery           exchange=celery(direct) key=celery
                

[tasks]
  . apps.analyses.tasks.perform_analysis

[2024-03-28 09:40:44,925: WARNING/MainProcess] /home/jan/PycharmProjects/truth_will_set_you_free/venv/lib/python3.12/site-packages/celery/worker/consumer/consumer.py:507: CPendingDeprecationWarning: The broker_connection_retry configuration setting will no longer determine
whether broker connection retries are made during startup in Celery 6.0 and above.
If you wish to retain the existing behavior for retrying connections on startup,
you should set broker_connection_retry_on_startup to True.
  warnings.warn(

[2024-03-28 09:40:44,930: INFO/MainProcess] Connected to redis://localhost:6379/0
[2024-03-28 09:40:44,931: WARNING/MainProcess] /home/jan/PycharmProjects/truth_will_set_you_free/venv/lib/python3.12/site-packages/celery/worker/consumer/consumer.py:507: CPendingDeprecationWarning: The broker_connection_retry configuration setting will no longer determine
whether broker connection retries are made during startup in Celery 6.0 and above.
If you wish to retain the existing behavior for retrying connections on startup,
you should set broker_connection_retry_on_startup to True.
  warnings.warn(

[2024-03-28 09:40:44,932: INFO/MainProcess] mingle: searching for neighbors
[2024-03-28 09:40:45,935: INFO/MainProcess] mingle: all alone
[2024-03-28 09:40:45,945: INFO/MainProcess] celery@jannone5 ready.
